images/checkpatch/README.md

```diff
@@ -46,3 +46,11 @@ When building the Docker image, several patches are applied to the script.
   contributed to the patch. Checkpatch understands this is some kind of tag,
   but not one it knows of, and it complains with a warning. This patch teaches
   it about the tag.
+
+## Custom Checks
+
+In addition to running `checkpatch.pl`, the bash script runs a few checks of
+its own on all commits (whether or not they touch the code under `bpf`).
```

Comment:
That means will need to rename the action in cilium/cilium, right?

Reply:
Because it's no longer just checkpatch.pl? We could, but the only additional check at this time is on commit subject length. This is very close to the other checks that checkpatch.pl runs, so I'm not sure it's worth a rename at this time.
Probably something we would do if we start extending with more, and more diverse, custom checks. I have no specific plan for other checks at the moment.

Or because that check applies to all commits? That's a good question. The majority of the checks are still run on commits touching bpf/ though, and the other jobs (coccicheck etc) are only for eBPF, so I'm not sure if renaming the action will make things any clearer.